text/0000-rfc-process.md

```diff
@@ -0,0 +1,238 @@
+* it impacts the spec.
+* it introduces a new component.
+* its implementation necessitates coordination across multiple teams.
+* it meaningfully impacts multiple personas (e.g. buildpack authors and platform authors).
+* the TOC requests that it be a project RFC.
+
+Given the nature of our project many RFCs will happen at the project level. However, some types of changes are more appropriately scoped to the team level. Some examples include:
+* Platform example: additions to or modification of the pack CLI interface (e.g. [pack pull policy](0046-pack-pull-policy.md)) or library interface (e.g. [pack logging refactor](0002-pack-logging-refactor.md)).
+* BAT example: a new major version of the libcnb API.
+* Implementation example: [Layer history](0102-history.md) or the lifecycle [multicall binary](0024-lifecycle-multicall-binary-for-build.md).
+* Distribution example: [Update CNB Registry JSON Schema](https://github.com/buildpacks/rfcs/pull/45).
+* Learning example: [intro video](0090-intro-video-script.md).
+
+### Process
+
+#### Requesting an RFC
+If you believe an RFC should be written but are either not prepared to author it personally or not prepared to do it _now_, please open an issue on this repo with a high level description of the desired change. This helps us keep track of good ideas and match make ambitious contributors with appropriately sized challenges.
+
+#### Drafting an RFC
+All RFCs begin life as a draft. Anyone wishing to propose a change to the project should create a draft RFC by:
+
+- Fork the RFC repo: <https://github.com/buildpacks/rfcs>
+- Copy `0000-template.md` to `text/0000-my-feature.md` (where 'my-feature' is descriptive. don't assign an RFC number yet).
+- Fill in RFC. Any section can be marked as "N/A" if not applicable.
+- Submit a pull request.
+
+#### Finding a Steward
+All RFCs, even project RFCs "belong" to a team. For project RFCs, The team lead of the responsible team is the steward of the RFC. For team RFCs any maintainer may be the steward. The author and the steward of an RFC may, at times, be the same person.
+
+In many but not all cases the appropriate team to own an RFC will be obvious. When the appropriate team is not obvious, the author should work with the community to find a home within one of the teams. Factors to consider when finding home include:
+* Which team has the most relevant technical context?
+* Which team has the deepest understanding of the use-case and the needs of impacted personas?
+* Which team is responsible for the components most impacted by the proposed change?
+* Which team is enthusiastic about supporting the change?
+
+If a home cannot be found for a draft RFC it remains in draft until one can be found. This does not necessary mean that the RFC is not a good idea or isn't something the project will take up eventually. It may simply be that the project does not have the bandwidth to prioritize this particular change at this particular time.
```

Comment:
```suggestion
If a home cannot be found for a draft RFC it remains in draft until one can be found. This does not necessarily mean that the RFC is not a good idea or isn't something the project will take up eventually. It may simply be that the project does not have the bandwidth to prioritize this particular change at this particular time.
```